### ISSUE TITLE
Kirby v3.5.3.1 throws navigation guard exception

### ISSUE THREAD

**Issue by GiantCrocodile** [getkirby/kirby#3186]
Hello,

when I delete a brand new draft, I get this exception in Firefox developer console:

```
Error: Redirected when going from "/pages/cwacwcwcaw" to "/pages" via a navigation guard.
```

This issue is introduced with v3.5.3.1.

**Console output**  

**Desktop (please complete the following information):**  
Latest firefox, Windows 10

**Screenshot**
![grafik](https://user-images.githubusercontent.com/6271057/109399455-2f674980-7943-11eb-92e1-635da7ddda38.png)

**Additional context**  
multilang site with just 1 language defined

Edit: OK, it just broke/hang because I had the checkbox 'Halt on exception' ticked; regardless of that, I didn't have this exception in the past so I'm not sure it's intended. Context of this error: https://stackoverflow.com/questions/62223195/vue-router-uncaught-in-promise-error-redirected-from-login-to-via-a

**Comment by lukasbestle** [comment]
@GiantCrocodile Which Kirby version did you use before the update? 3.5.3.1 didn't have any changes in the Vue routing, so maybe you are seeing the effects of a prior release?

**Comment by GiantCrocodile** [comment]
@lukasbestle Hi! Before I did use Kirby v3.5.2, then v3.5.3 and now 3.5.3.1. I upgraded them some days ago due to fixes I were interested in. I think in v3.5.2 this was no issue for sure so maybe 3.5.3?

**Comment by texnixe** [comment]
@GiantCrocodile Could you please provide step-by-step instructions how to reproduce this in a fresh Starterkit?

**Comment by bastianallgeier** [comment]
It's important to note, that those exceptions are "just" a stupid design decision in Vue router and don't have any negative side effects. They should not be an error in the first place. I still wonder why this one turns up. I thought we already took care of them.

**Comment by distantnative** [comment]
I can only imagine that it's a slightly different error than `NavigationDuplicated` which we try to catch. So stupid from `vue-router` to throw errors for actions that are supported and work just fine.

@GiantCrocodile I guess the question is to which page you got redirected instead `/pages`, login maybe? Cause as far as I understand this error of `vue-router` is basically complaining that we routed from `/pages/cwacwcwca` to `/pages` but then a navigation guard added another redirect (basically two subsequent redirects in one navigation action).

**Comment by distantnative** [comment]
✅ Solved with upcoming 3.6.0 (since we are moving away from `vue-router`)
